stdlib/public/SDK/Foundation/Foundation.swift

```diff
@@ -567,11 +567,12 @@ extension Array : _ObjectiveCBridgeable {
 
 extension NSDictionary : ExpressibleByDictionaryLiteral {
   public required convenience init(
-    dictionaryLiteral elements: (NSCopying, AnyObject)...
+    dictionaryLiteral elements: (_Hashable, Any)...
```

Comment:
@gribozavr Since we coerce to `AnyObject` then dynamically check `NSCopying` anyway, there's probably little reason not to use `Any` for the literal key type, if you prefer that.